File: primerTrimestre/java basico/1secuenciales/Ej15sec.java
/**Dadas dos variables numéricas A y B, que el usuario debe teclear, se pide realizar un algoritmo
 * que intercambie los valores de ambas variables y muestre cuanto valen al final las dos variables.
 * 
 * @author Antonio Quesada
 */


import java.util.Scanner;

public class Ej15sec {
  public static void main(String[] args) {
    Scanner s = new Scanner(System.in);
    
    //Definimos las variables.
    int a;
    int b;
    int aux;
    
    //Pedimos un valor de a y b.
    System.out.println("Introduce el valor de a: ");
    a = s.nextInt();
    System.out.println("Introduce el valor de b: ");
    b = s.nextInt();
    
    //Intercambiamos el valor de cada variable usando una variable auxiliar.
    aux = a;
    a = b;
    b = aux;

    //Mostramos el nuevo valor de a y b.
    System.out.println("El nuevo valor de a es "+a);
    System.out.println("El nuevo valor de b es "+b);
  }

}

File: primerTrimestre/introduccion_programacion_python/1secuenciales_py/ej15sec.py
# Programa: ej15sec.py
# Propósito: Dadas dos variables numéricas A y B, que el usuario debe teclear, se pide realizar un algoritmo
# que intercambie los valores de ambas variables y muestre cuanto valen al final las dos variables.
#
# Autor: Antonio Quesada
# Fecha: 13/10/2019.
#
# Variables a usar:
#   * a
#   * b
#   * aux
#
# Algoritmo:
#   LEER a, b, aux
#   aux = a
#   a = b
#   b = aux
#   ESCRIBIR a
#   ESCRIBIR b

# Petición de datos
a = float(input("Introduce el valor de la variable a: "))
b = float(input("Introduce el valor de la variable b: "))
print("-------------------------------------------")

#Proceso
aux = a
a = b
b = aux

#Salida
print("El nuevo valor de a es", a)
print("El nuevo valor de b es", b)
  




	
	

	






	
	

	
